contracts/PetPark.sol

```diff
@@ -1,7 +1,65 @@
 //SPDX-License-Identifier: Unlicense
 pragma solidity ^0.8.0;
 
-
 contract PetPark {
 
+    address public owner;
+    mapping(uint => uint) public animalCounts;
+
+    uint private stable_age;
```

Comment:
Since you need to store these for multiple users, better to use a struct here to combine a person's profile and map them to addresses